dev-python/subprocess32/files/subprocess32-3.2.7-sandbox-test-fix.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/test_subprocess32.py b/test_subprocess32.py
+index c312949..d519aef 100644
+--- a/test_subprocess32.py
++++ b/test_subprocess32.py
+@@ -538,7 +538,7 @@ class ProcessTestCase(BaseTestCase):
+                 [sys.executable, "-c",
+                  'import os; '
+                  'print([k for k in os.environ.keys() '
+-                 '       if ("VERSIONER" not in k and "__CF" not in k)])'],
```

Comment:
Why are you removing those variables? I suppose this would break the test on OSX Prefix for no good reason.